### ISSUE TITLE
Narator does not read the value of  `allowInsecureConnections` 

### ISSUE THREAD

**Issue by Nigusu-Allehu** [NuGet/Home#13555]
How to replicate

1. create dotnet project
2. add a nuget.config file with an http source
3. set the http source allowInsecureConnections value to true
4. Now press ctrl+windows+enter to start narrator
5. Open PMUI and select the http package source
6. Listen to the narrator, it reads a warning even though there should not be a warning.

This is caused because the allowInsecureConnections value is not propagated down to the accessibility object
